### PR TITLE
molten-vk 1.1.11

### PR DESCRIPTION
**Pull request by rcombs**
Created with `brew bump-formula-pr`.

Reordered resource blocks for easier upgrading in the future.

### CODE REVIEW

**Comment by Porkepix** [comment]
Resources needs to be updated, also I proposed to reorder them to make it easier here: https://github.com/Homebrew/homebrew-core/pull/108057
That wasn't enough though and the build is broken as-is.

**Comment by rcombs** [comment]
Yup, fixing now (and I'll go ahead and reorder as well while I'm at it); didn't realize that running the command would create the PR without giving me a chance to verify everything looked good first.

**Comment by Porkepix** [comment]
> Yup, fixing now (and I'll go ahead and reorder as well while I'm at it); didn't realize that running the command would create the PR without giving me a chance to verify everything looked good first.

Depends if you use `brew bump` or `brew bump-formula-pr`. The former calls the latter. The latter have options to only write changes locally and don't open the PR (with `--write-only`, I believe also `-w`, not sure though), though one can want to just have the PR be created and then edit it. Ideally an option to do the whole process with a way to edit it in-between would be even better.

**Comment by rcombs** [comment]
This is now updated with all the correct resource-block changes, along with the build-procedure changes needed for the new version. Builds locally and passes CI.

File: Formula/molten-vk.rb [hunk 1]
```diff
@@ -2,55 +2,54 @@ class MoltenVk < Formula
...
     # For each resource the path to the file describing the expected
     # revision is listed.
-    resource "cereal" do
-      # ExternalRevisions/cereal_repo_revision
-      url "https://github.com/USCiLab/cereal.git",
-          revision: "51cbda5f30e56c801c07fe3d3aba5d7fb9e6cca4"
+    resource "SPIRV-Cross" do
+      # ExternalRevisions/SPIRV-Cross_repo_revision
+      url "https://github.com/KhronosGroup/SPIRV-Cross.git",
+          revision: "61c603f3baa5270e04bcfb6acf83c654e3c57679"
```
Comment (chenrui333) [inline Formula/molten-vk.rb:16]: just curious, why is there a resource order change in here? Thanks!

Comment (rcombs) [reply Formula/molten-vk.rb:16]: With this change, all resources are in the order they're listed in the repositories the revisions come from:
https://github.com/KhronosGroup/MoltenVK/tree/v1.1.11/ExternalRevisions
https://github.com/KhronosGroup/glslang/blob/73c9630da979017b2f7e19c6549e2bdb93d9b238/known_good.json

This makes it easier to just go down the list and grab the new revisions.

**Review by carlocab — APPROVED** [review]
Thanks!

File: Formula/molten-vk.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/molten-vk.rb:16]: make sense, thanks!

**Pull request closed without merging**